cli.ts

```diff
@@ -705,6 +748,11 @@ const sanitizeArgs = (parsedArgs: RawInitArgs) : SanitizedInitArgs => ({
     setVersion: parsedArgs.setVersion,
     version: parsedArgs.version
 })
+const sanitizeScaffoldArgs = (parsedArgs: RawInitArgs) : SanitizedScaffoldInitArgs => ({
```

Comment:
Relates to https://github.com/ecadlabs/taqueria/pull/139/files#r810054510

Reply:
Ok, will comment there